<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_debug</class>
 <widget class="QDialog" name="Dial_debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>466</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dial_debug</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addr">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Addr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_wr_dat">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButt_wr">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Rd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_count">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>171</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_rd_dat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rej">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>317</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DLC">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dlc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DLC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_data">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send_can">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>244</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_dat0">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat1">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat2">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat3">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat4">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat5">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat6">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dat7">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dial_debug</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dial_debug</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dial_debug</receiver>
   <slot>slot_butt_rd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButt_wr</sender>
   <signal>clicked()</signal>
   <receiver>Dial_debug</receiver>
   <slot>slot_butt_wr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>Dial_debug</receiver>
   <slot>clear_textEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_send_can</sender>
   <signal>clicked()</signal>
   <receiver>Dial_debug</receiver>
   <slot>slot_butt_wr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_butt_rd()</slot>
  <slot>slot_butt_wr()</slot>
  <slot>clear_textEdit()</slot>
 </slots>
</ui>
